docs/self-hosted-appcircle/configure-server/advanced-configuration/store-dist-dmz.md

```diff
@@ -0,0 +1,503 @@
+---
+title: Enterprise App Store and Testing Distribution in DMZ
+description: Learn how to create a server in DMZ for Appcircle Enterprise App Store and Testing Distribution for accessing from internet.
+tags:
+  [
+    self-hosted,
+    advanced configuration,
+    dmz,
+    enterprise app store,
+    testing distribution,
+  ]
+sidebar_position: 12
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import SpacetechExampleInfo from '@site/docs/self-hosted-appcircle/configure-server/\_spacetech-example-info.mdx';
+import LingerOption from '@site/docs/self-hosted-appcircle/configure-server/\_linger-option.mdx';
+import SocatConfiguration from '@site/docs/self-hosted-appcircle/configure-server/\_socat-configuration.mdx';
+import NetavarkConfiguration from '@site/docs/self-hosted-appcircle/configure-server/\_podman-netavark-configuration.mdx';
+import FirewalldConfiguration from '@site/docs/self-hosted-appcircle/configure-server/\_firewalld-configuration.mdx';
+import UFWConfiguration from '@site/docs/self-hosted-appcircle/configure-server/\_ufw-configuration.mdx';
+import SwapConfiguration from '@site/docs/self-hosted-appcircle/configure-server/\_swap-configuration.mdx';
+import DowntimeCaution from '@site/docs/self-hosted-appcircle/configure-server/\_appcircle-server-downtime-caution.mdx';
+
+## Overview
+
+A Demilitarized Zone (DMZ) in networking is a segment of an internal network that is exposed to external networks, typically over the internet. By isolating Appcircle DMZ server from the internal network, you ensure that it remains secure.
+
+This is particularly useful when users need to access Testing Distribution and Enterprise App Store but not all features for business operations within the private network.
+
+The Testing Distribution module and Enterprise App Store module hosted on the Appcircle DMZ server can be accessed by users from the internet, ensuring they have secure access to these critical features while keeping sensitive business data within your private network. This setup provides a balance between security and productivity in an organization's IT environment.
+
+@TODO: Diagram will be here.
+
+We assume that you have already set up an Appcircle server successfully. This document will guide you through creating Appcircle DMZ server and Appcircle server configurations.
+
+In this document:
+
+- We will call the "Appcircle DMZ server" to the server, which is located in the DMZ and host the Appcircle Enterprise App Store and Testing Distribution services.
+
+- We will call the "Appcircle server" to the server, which is located in the private network and host the Appcircle core services.
+
```

Comment:
We should have an info box that explains user that when you switch to the DMZ architecture both Enterprise App Store and Testing Distribution will be moved to the DMZ architecture. Currrenly, we do not  support using only one of them in the DMZ.

This is a known limitation.